src/submit_jobs.py

```diff
@@ -122,9 +132,12 @@ def submit_jobs(
 
     # Each session is shared across all read jobs and manages job order,
     # credentialing, retries, etc.
+    target = "TST" if extract_target == "test" else "PRD"
```

Comment:
I don't _love_ just adding simple conditional logic inline like this, but I also don't expect that we'll need to target environments any time soon. So, IMO it's fine for now.

Reply:
**[Praise]** Agreed! I also don't think it'll be too difficult to refactor in the future if this assumption changes.